test/fixtures/profiles/inputs/cli/controls/cli.rb

```diff
@@ -8,3 +8,65 @@
     it { should cmp "value_from_cli_02"}
   end
 end
+
+control "test_control_numeric_implicit" do
+  describe input("test_input_03", value: "value_from_dsl") do
+    it { should eq 11 } # Note eq not cmp
```

Comment:
We could maybe include assertions like `its('class') { should eq Integer }` because `11` and `11.0` are different classes but return true for `eq`